azavea/pwd-fishway
# Bring in CA app template

## Overview

Sets up the project with the CA app template in preparation for development. We are on a tight project timeline and setting this work up now saves development time down the road while we are still in the design phase.

I committed to `develop` the PR template, and issue templates.

Connects #1

### Demo

<img width="897" alt="Screen Shot 2019-03-26 at 4 52 43 PM" src="https://user-images.githubusercontent.com/10568752/55033551-d0c42300-4fe9-11e9-9b77-159772b1420d.png">



## Testing Instructions
A default CRA app should display at http://localhost:3474 after VM setup and starting the server

`./scripts/setup`
`vagrant ssh`
`./scripts/server`
